www/api/internal.php

```diff
@@ -45,7 +45,7 @@
 
 $pearDB = new CentreonDB();
 
-CentreonSession::start(1);
+CentreonSession::start();
```

Comment:
any reason to make this change ?

Reply:
with 1, it opens and close juste after the session. you cannot store in session.

Reply:
Bad practice because the first call will lock the session in writing and all other calls will be put on hold.
The good practice will be to open it if necessary and close it immediately afterwards.

Reply:
so you want me to reopen the session for each $_SESSION changes ?

Reply:
Just open the session before to update it and after the changes you close it immediately without waiting that the end of the script do it for you.

Reply:
For example in your case, open the session and read the parameters that you need at the start and close the session. Next, reopen it before to make change on the session and close it just after.

Reply:
I'll add an method for that

Reply:
I have the done patch